src/legacy/core_plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -46,7 +46,6 @@ import {
 import { dispatchRenderComplete } from '../../../../../plugins/kibana_utils/public';
 import { SavedSearch } from '../../../kibana/public/discover/np_ready/types';
 import { Vis } from '../np_ready/public';
-import { queryGeohashBounds } from './query_geohash_bounds';
```

Comment:
Can we delete the `query_geohash_bounds` file then? I think this was the only place it was used.